projects/batfish-client/src/main/java/org/batfish/client/BfCoordWorkHelper.java

```diff
@@ -389,27 +389,32 @@ private ClientBuilder getClientBuilder() throws Exception {
 
```

Comment:
looks like your google-java-format plugin is messed up. Please ensure it's installed. If it's acting wonky, maybe disable and renable?